cmd/virt-chroot/tap-device-maker.go

```diff
@@ -16,18 +16,14 @@ func createTapDevice(name string, owner uint, group uint, queueNumber int, mtu i
 		LinkAttrs:  netlink.LinkAttrs{Name: name},
 		Mode:       unix.IFF_TAP,
 		NonPersist: false,
-		Queues:     queueNumber,
 		Owner:      uint32(owner),
 		Group:      uint32(group),
 	}
 
-	// when netlink receives a request for a tap device with 1 queue, it uses
-	// the MULTI_QUEUE flag, which differs from libvirt; as such, we need to
-	// manually request the single queue flags, enabling libvirt to consume
-	// the tap device.
-	// See https://github.com/vishvananda/netlink/issues/574
```

Comment:
The comment is still relevant per what I can see. Can you explain why you think it is no longer relevant?

Reply:
Because we no longer pass the `Queues` field to `github.com/vishvananda/netlink`.

Reply:
The condition is kept and the body is kept, both are not part of the comment.
The input is determining the flag used and that is what the comment is talking about.

Reply:
Ok, so adjust it then to reflect the same decision which you take now in other words. I.e. the decision to touch the flag explicitly is the same, to mimic how libvirt behaves and assure it is in sync with it.

Reply:
Done.